geschaeftsordnung-vollversammlung.md

```diff
@@ -0,0 +1,82 @@
+# Geschäftsordnung der Vollversammlung der Fachschaft Digital Engineering
+
+Diese Geschäftsordnung wurde am xx.xx.xxxx von der Vollversammlung beschlossen.
+
+## § 1 Geltungsbereich
+
+Die Geschäftsordnung gilt für die Vollversammlung der Fachschaft Digital Engineering an der Universität Potsdam.
+
+## § 2 Tagesordnung
+
+(1) Die Vollversammlung folgt einer Tagesordnung.
+
+(2) Die Tagesordnung muss Teil der fristgerechten Einladung einer Vollversammlung sein.
+
+(3) Bis 14 Tage vor einer Vollversammlung können die einladenden Personen gestellte Anträge in die Tagesordnung aufnehmen.
+
+(4) Bis 7 Tage vor einer Vollversammlung muss die finale Tagesordnung auf dem gleichen Weg wie die Einladung zur Vollversammlung bekanntgegeben werden.
+
+## § 3 Versammlungsleitung
+
+(1) Die Vollversammlung wird von den einladenden Personen geleitet.
+
+(2) Die Versammlungsleitung kann für einzelne Tagesordnungspunkte an andere Mitglieder der Fachschaft übergeben werden.
+
+(3) Die Versammlungsleitung kann bei Bedarf eine Redeliste führen und/oder die Redezeit begrenzen.
+
+(4) Die Versammlungsleitung entscheidet, auf welche Weise Stimmen gezählt oder geschätzt und ob sie anschließend protokolliert werden.
+
+## § 4 Protokolle
+
+(1) Die Vollversammlung dokumentiert ihre Beschlüsse in einem Protokoll.
+
+(2) Die Versammlungsleitung beauftragt ein Mitglied der Fachschaft mit der Protokollführung.
+
+(3) Das Protokoll wird der Fachschaft spätestens 7 Tage nach der Vollversammlung zugänglich gemacht.
+
+## § 5 Anträge
+
+(1) Anträge sind schriftlich zu verfassen und innerhalb von 14 Tagen nach Zugang der Einladung zur Vollversammlung bei den einladenden Personen einzureichen.
+
+(2) Die einladenden Personen entscheiden über die Aufnahme des Antrags in die Tagesordnung.
+
+(3) Anträge können vor der Behandlung durch die antragstellende Person zurückgezogen werden.
+
+## § 6 Änderungsanträge
+
+(1) Änderungsanträge werden vor der Abstimmung des jeweiligen Antrags behandelt.
+
+(2) Änderungsanträge können von der den ursprünglichen Antrag stellenden Person direkt angenommen werden. Geschieht dies nicht, stimmt die Vollversammlung über die Annahme ab.
+
+(3) Änderungsanträge können vor der Vollversammlung schriftlich bekanntgegeben und begründet werden.
+
+## § 7 Anträge zur Geschäftsordnung
+
+(1) Anträge zur Geschäftsordnung werden durch das Heben beider Arme angezeigt. Davon kann abgewichen werden, wenn die Intention für die Versammlungsleitung eindeutig erkennbar ist.
```

Comment:
Wenn wir eine spezifische Meldung für  GO-Anträge definieren sollten wir das auch für Diskussionsbeiträge und Änderungsanträge machen. Ich kann mir aber auch sehr gut vorstellen, dass das in der Ordnung selbst nicht nötig ist, sondern einfach per Konvention gelernt wird.

Reply:
Sehe ich eher nicht so. GO-Anträge haben ja höchste Priorität und werden u.U. sogar während einer Wortmeldung behandelt (wenn Antrag auf sofortige Beendigung der Diskussion). Diskussionsbeiträge/ÄA können mMn in normaler Reihenfolge - First comes, first serves - behandelt werden und erfordern eben keine besondere Priorisierung. 

Das Verfahren ist dann halt so:

> Sitzungsleitung: Antrag X über Y wird jetzt behandelt. Gibt es Diskussionsbeiträge oder Änderungsanträge?
> A: Redebeitrag
> B: Redebeitrag
> C: Änderungsantrag --> Abstimmung, evtl. Änderung des Antrags
> D: Redebeitrag
> E: Änderungsantrag --> Abstimmung, evtl. Änderung des Antrags
> F: Redebeitrag
> Ende der Redeliste, Abstimmung über Antrag in Fassung von E

Reply:
Mit dieser Definition wollen wir verhindern, dass sich Menschen mit GO-Anträgen "normal" melden und dadurch nicht priorisiert behandelt werden können. Das halten wir für so wichtig, dass das nicht per Konvention gelernt werden müssen sollte. Der zweite Satz ist einmal online sehr wichtig, aber auch in Präsenz ist es uns relativ egal, ob jemand zwei Arme hebt oder winkt und "ich habe einen GO-Antrag" ruft.
Anders herum sehe ich keinen Bedarf an der Definition einer normalen Meldung, da diese auf jede beliebige Art geschehen kann, solange sie von der Sitzungsleitung bemerkt wird

Reply:
@ADimeo Kann dieser Thread geschlossen werden?